### PR TITLE
fix: typo kafta -> kafka

### PR DESCRIPTION
**Pull request by aaron-trout**

### CODE REVIEW

**Comment by aaron-trout** [comment]
Regarding the CLA... I would have to submit a request to my company's legal team to get approval to sign it. Really not worth it for a single character PR so best would be to close this PR and just fix it yourselves :-)

**Review by x8a — APPROVED** [review]
Thanks for spotting the typo, @aaron-trout!

So would you rather us close the PR or we wait for your company's legal team response?

**Comment by x8a** [comment]
I'll close the PR and fix the typo. Thanks!

**Pull request closed without merging**
